### DISCUSSION TITLE
Wheel drilling jig

### DISCUSSION THREAD

**Discussion by mredmon63** [nasa-jpl/open-source-rover#266] [Q&A] [answered]
Please point me to the STL for a wheel drilling jig.
Thanks,
Mark

**Comment by JHPHELAN** [comment, marked as answer]
[https://www.dropbox.com/s/40tviy76ktb91hm/wheel_adapter3.STL?dl=0](url)
@mredmon63 Don't know where it disappeared to.  But here ya go!  Happy building!

**Reply by apollokit** [reply]
Hmm, @JHPHELAN that link isn't working for me...

We should definitely get this included in the documentation. I'd say we just put the stl file in the repo. I'll open an issue.

**Reply by JHPHELAN** [reply]
I can't get link to work either ?!?
How shall I upload it to github?

**Reply by apollokit** [reply]
well....where did you get that link in the first place? :)

Don't worry about it, I'll handle it - once I know where the jig file is.  Issue: https://github.com/nasa-jpl/open-source-rover/issues/270

**Reply by JHPHELAN** [reply]
@apollokit I have the .stl file.  Sometime in the past I uploaded it somewhere to the community, either github or TapaTalk, but can't find it now.  TapaTalk won't take a .stl file.  So I uploaded it to DropBox and shared the link.  But then the link doesn't work?!
How can I re-upload to the project?
JHP

**Reply by apollokit** [reply]
Ok so the situation is:
- you have the file
- you're apparently able to upload it to dropbox
- but the dropbox link you shared doesn't work

Seems like the easiest thing to do is figure out why your dropbox link above didn't work, fix that, and then repost that link here

@JHPHELAN

**Reply by Achllle** [reply]
The file is in #46 ;)
